### PR TITLE
Don't check removed transcriptions

### PR DESCRIPTION
**Pull request by itsthejoker**
Relevant issue: https://github.com/GrafeasGroup/tor/pull/215

## Description:

The other side of the linked PR. This changes the posts to Slack so that removed transcriptions are never brought up checking.

## Checklist:

- [x] Code Quality
- [x] Pep-8
- [x] Tests (if applicable)
- [x] Success Criteria Met
- [ ] Inline Documentation
- [ ] Wiki Documentation (if applicable)

### CODE REVIEW

**Review by MaxVanDeursen — APPROVED** [review]

**Comment by itsthejoker** [comment]
After thinking about it, I'm not happy about this yet. We know from experience that a lot of new volunteers have their transcriptions nuked by the spam filter, and with this as it's written right now, it's possible to skip their first transcription getting posted if it's removed by Reddit. I'm going to rework this a little bit.

**Comment by TimJentzsch** [comment]
What about posting the transcription text in a code block instead? Once it's marked as `done` Blossom should have that data, right?
Could spam the channel a bit, but if it's only for the removed transcriptions it's probably fine.

**Comment by itsthejoker** [comment]
I forgot this was a thing. Will address.

**Comment by itsthejoker** [comment]
I'm only commenting to get the github bot to shut up for a little bit. Currently working on https://github.com/GrafeasGroup/blossom/pull/188

File: api/views/submission.py [hunk 1]
```diff
@@ -399,9 +399,12 @@ def done(self, request: Request, pk: int, username: str = None) -> Response:
...
             if self._should_check_transcription(user):
-                self._send_transcription_to_slack(
-                    transcription, submission, user, slack
-                )
+                # Check to see if the transcription has been removed. If it has, only
+                # post the message to slack if the user has completed 5 or fewer posts.
+                if not transcription.removed_from_reddit or user.gamma <= 5:
+                    self._send_transcription_to_slack(
+                        transcription, submission, user, slack
+                    )
```
Comment (TimJentzsch) [inline api/views/submission.py:407]: Our `done` logic is getting quite complex, I wonder if we can simplify it somehow. But that's something for another PR.

**Review by TimJentzsch — APPROVED** [review]

**Pull request merged**